test/test_layouts.jl

```diff
@@ -176,26 +176,45 @@ struct FooNumber <: Number end
         @test symmetricdata(Symmetric(transpose(B))) ≡ transpose(B)
         @test hermitiandata(Hermitian(transpose(B))) ≡ transpose(B)
 
-        @testset "Bidiagonal" begin
-            B = Bidiagonal(randn(6),randn(5),:U)
-            Bc = Bidiagonal(randn(6) .+ 0im,randn(5) .+ 1im,:U)
-            S = Symmetric(B)
-            H = Hermitian(B)
-            Sc = Symmetric(Bc)
-            Hc = Hermitian(Bc)
+    end
+
+    @testset "Symmetric of Banded" begin
```

Comment:
I don't think the banded tests are needed here since the `Bidiagonal` tests below would trigger the same codepath.

Reply:
> the Bidiagonal tests below would trigger the same codepath.

I tried that first - without success because of `SymTridiagonalLayout` behaves differently. I use a test type to overcome the circularity.